docs/data-access/icgc-25k-data.md

```diff
@@ -0,0 +1,100 @@
+
+## Accessing ICGC 25K Release Data
+
+An SFTP server is available to access ICGC Release Data and PCAWG data.
+
+The server hosts three data directories with the following data:
+
+- `/release_28` - This is the Data Portal data Release 28 of the International Cancer Genome Consortium (ICGC).
+- `/PCAWG` - Analysis results from the PCAWG study.
+- `/Supplemental` - Corrected clinical metadata and RNA-Seq raw read counts for projects LICA-FR and PRAD-UK.
+
+Only users with DACO approval are able to access the SFTP server. If you previously had DACO access for ICGC 25K data you will continue to have permission to access the SFTP server. If you require DACO approval please see the documentation on [applying for DACO access](./daco/applying.md).
+
+### SFTP Connection Details
+
+The SFTP server is located at:
+
+- **Host**: `legacy-icgc-sftp.platform.icgc-argo.org`
+- **Port**: `2222`
+
+Authentication to the server is done using username and password:
+
+- **Username**: The email address that was approved for DACO access. This is the account you would use to log into the [ARGO platform](https://platform.icgc-argo.com).
+- **Password**: ICGC API Key. This is available on your ARGO Platform [profile page](https://platform.icgc-argo.com/user).
+
+You can connect to this server using any SFTP client of your choice. For a free client you consider [FileZilla](https://filezilla-project.org/download.php?type=client).
+
+## Partner Repositories with ICGC 25K File Data
+
+ICGC file data is hosted across the following repositories.
+
+If you know the specific files from ICGC that you want to access, consult the provided [Mapping File](#mapping-icgc-legacy-data-to-external-repositories) to discover which repositories host that file.
+
+### EGA
+
+Raw sequences submitted to ICGC or reprocessed apart of PCAWG are hosted across multiple EGA datasets managed by the data access committee [EGAC00001000010](https://ega-archive.org/dacs/EGAC00001000010).
+
+Upon DACO approval, the institutional email (the same email used in your DACO application, not the affiliated Gmail account) can be used to log in/sign up at [EGA-Archive](https://ega-archive.org).
+
+If the email has never been used to access EGA, please follow the password reset procedure.
+
+Access to EGA may take up to 48 hours post DACO approval.
+
+### PDC
+
+Due to data regulation policies, TCGA affiliated data from the PCAWG study are saved in PDC and controlled under dbGap study [phs000178](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000178.v11.p8).
+
+DACO approval does not include dbGap study [phs000178](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000178.v11.p8) and will not grant access to said files.
```

Comment:
ICGC 25k DACO approval does not....

Reply:
Updated:
https://github.com/icgc-argo/argo-docs/blob/83f642c19a1835dc57dd745206451e6c41e8293a/docs/data-access/icgc-25k-data.md?plain=1#L69